xup/xup.c

```diff
@@ -212,7 +212,7 @@ lib_mod_connect(struct mod *mod)
         if (trans_connect(mod->trans, mod->ip, con_port, 3000) == 0)
         {
             LLOGLN(0, ("lib_mod_connect: connected to Xserver "
-                   "(Xorg or X11rdp) sck %ld", mod->trans->sck));
+                   "(Xorg or X11rdp) sck %ld", (long) (mod->trans->sck)));
```

Comment:
I think it's that way because SOCKET on Windows is defined as a HANDLE which is void*
I think it's really just an int and this is a strange Microsoft thing.  I think openssl on Windows just cast it to an int anyway.  Lets not change this now as it's always big enough and this PR is just for warnings.